lib/classes/CLI.js

```diff
@@ -9,7 +9,7 @@ const chalk = require('chalk');
 class CLI {
   constructor(serverless, inputArray) {
     this.serverless = serverless;
-    this.inputArray = (typeof inputArray !== 'undefined' ? inputArray : []);
+    this.inputArray = inputArray || undefined;
```

Comment:
Would be better to use `null` here because `undefined` is a more JavaScript engine specific feature which a variable shouldn't be set to.